src/controllers/authentication/session.rs

```diff
@@ -83,3 +88,8 @@ pub fn build_message_framework(signing_key: Key) -> FlashMessagesFramework {
     let message_store = CookieMessageStore::builder(signing_key).build();
     FlashMessagesFramework::builder(message_store).build()
 }
+
+#[derive(Deserialize, Debug, Clone, PartialEq, sqlx::FromRow, Default)]
+pub struct Password {
+    pub password: String,
+}
```

Comment:
This is supposed to be defined in `model` but not `controller` ?

Reply:
fixed it